app/controllers/custom_fields_controller.rb

```diff
@@ -63,9 +63,16 @@ def create
   def edit; end
 
   def update
+    last_update_before = @custom_field.custom_options.map(&:updated_at).max
     @custom_field.attributes = get_custom_field_params
 
     if @custom_field.save
+      last_update_after = @custom_field.custom_options.map(&:updated_at).max
+
+      if last_update_after.try { |date| date > (last_update_before || date) }
+        @custom_field.touch # touch to invalidate cache
+      end
```

Comment:
What you have build here should have been done by https://github.com/opf/openproject/blob/dev/app/models/custom_option.rb#L37 already. So the question is: Why didn't the standard way work?